pages/graphs/[name].tsx

```diff
@@ -90,7 +90,7 @@ const CustomTooltip = ({ payload, label, content }: any) => {
         {label.split("T")[0].slice(2)}
       </Box>
       <Box fontFamily="" fontSize="2xl" fontWeight="700">
-        {payload[0]?.value.toFixed(2)}
+        {parseInt(payload[0]?.value).toFixed(2)}
```

Comment:
Still need to use consistent method/format for fetching data and passing it to components, eg: `getServerSideProps` vs client-side fetching AND need to figure out how to use meaningful types for page data props, and not rely on JSON strings.

Reply:
Resolve in #8